build.rs

```diff
@@ -0,0 +1,12 @@
+use std::process::Command;
+
+fn main() {
+    println!("cargo:rustc-env=GIT_COMMIT_HASH={}", "");
+    match Command::new("git").args(&["rev-parse", "HEAD"]).output() {
+        Ok(output) => match String::from_utf8(output.stdout) {
+            Ok(hash) => println!("cargo:rustc-env=GIT_COMMIT_HASH={}", hash),
```

Comment:
`"cargo:rustc-env=GIT_COMMIT_HASH={}"` is a magic value. Please extract it to a properly named variable.

Reply:
It's syntax used to set an env!() value during builds.

https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env

<img width="825" alt="image" src="https://github.com/user-attachments/assets/b3e8ef2f-7bd6-45e0-b3f2-2e5a68be8e5e">

Reply:
Would you consider this an improvement?
<img width="784" alt="image" src="https://github.com/user-attachments/assets/743388d4-9ed4-4338-959f-89244f28651f">

Reply:
> It's syntax used to set an env!() value during builds.

In that case, it can stay that way.